HHS/Head-Start-TTADP
# [TTAHUB-1693] Fix display of start date in filter when pasted into a browser

## Description of change
This PR fixes a bug where the filter pill would display an incorrect label when the URL was pasted into a web browser.

## How to test
- visit the regional dashboard or the activity reports landing page
- Filter - Select region = 3
- Filter - Select date started is year to date. 
- Copy URL into Slack channel and post it.
- Click on hyperlink in Slack.
- Confirm date is rendered correctly

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1693


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
